Objects/typeobject.c

```diff
@@ -402,33 +403,90 @@ _PyType_Name(PyTypeObject *type)
     return s;
 }
 
+
 static PyObject *
-type_name(PyTypeObject *type, void *context)
+_PyType_QualName(PyTypeObject *type)
```

Comment:
Why `type_qualname()` and `type_module()` have been renamed to `_PyType_QualName()` and `_PyType_Module()`?

Reply:
Hum. I moved code in a weird way. I fixed that.